modules/nf-core/odgi/viz/tests/main.nf.test.snap

```diff
@@ -24,10 +24,14 @@
                     ]
                 ],
                 "versions": [
-                    "versions.yml:md5,95c0ee8015cf342c2724fb64d4cba67e"
+                    "versions.yml:md5,d819e8f806b414b178c2d915e9d9791e"
                 ]
             }
         ],
-        "timestamp": "2024-01-30T09:53:51.373368285"
+        "meta": {
+            "nf-test": "0.9.2",
+            "nextflow": "24.10.3"
+        },
+        "timestamp": "2025-01-15T14:50:21.102589"
     }
 }
```

Comment:
cause EOcmake -H. -Bbuild && cmake --build build -- -j 3L is everywhere in each snap